### PR TITLE
WINDUP-4225: MTR End of Life Admonition Added

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [WINDUP-4225](https://issues.redhat.com/browse/WINDUP-4225)

### PREVIEW

* [CLI guide](https://anarnold97.github.io/MTA-Preview/MTR/end-of-life/cli-guide.html#introduction)
* [Eclipse guide](https://anarnold97.github.io/MTA-Preview/MTR/end-of-life/eclipse-guide.html#introduction_eclipse-code-ready-studio-guide)
* [Getting started guide](https://anarnold97.github.io/MTA-Preview/MTR/end-of-life/getting-started-guide.html#introduction)
* [IntelliJ guide](https://anarnold97.github.io/MTA-Preview/MTR/end-of-life/intellij-idea.html#introduction)
* [Maven guide](https://anarnold97.github.io/MTA-Preview/MTR/end-of-life/maven-guide.html#introduction)
* [Rules development guide](https://anarnold97.github.io/MTA-Preview/MTR/end-of-life/rules-development.html#introduction_rules-development-guide-mtr)
* [VS Code guide](https://anarnold97.github.io/MTA-Preview/MTR/end-of-life/vs-code-guide.html#introduction)
* [Web-console guide](https://anarnold97.github.io/MTA-Preview/MTR/end-of-life/webconsole-guide.html#introduction)

|       |
| ---- |
| image |

### CODE REVIEW

**Comment by anarnold97** [comment]
@kpunwatk and @PhilipCattanach - Adding the admonition

We will also add a banner to the documentation at a later date

Thanks

File: docs/topics/snippet-mtr-end-of-life.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+:_content-type: SNIPPET
+
+[IMPORTANT]
+.{ProductName} product will be End of Life on September 30th, 2024
```
Comment (anarnold97) [inline docs/topics/snippet-mtr-end-of-life.adoc:4]:
```suggestion
.{ProductName} product will be End-of-Life on September 30th, 2024
```

Or does it look better hyphenated?

**Comment by kpunwatk** [comment]
> @kpunwatk and @PhilipCattanach - Adding the admonition
> 
> We will also add a banner to the documentation at a later date
> 
> Thanks

Looks good to me, Andy. Additionally, the hyphenated version "End-of-Life" looks perfect. Thanks!

**Review by kpunwatk — APPROVED** [review]

**Comment by PhilipCattanach** [comment]
That looks good to me - thanks @anarnold97

**Comment by raoradhika** [comment]
PR merged

**Pull request merged**